ast/src/main/java/com/graphicsfuzz/common/glslversion/Glsl330.java

```diff
@@ -34,7 +34,7 @@ public String getVersionString() {
   public boolean supportedDeterminant() {
     // According to this page:
     // https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/determinant.xhtml
-    // determinant should be supported from GLSL 1.40, but glslangValidator disagrees.
```

Comment:
Can you double-check this?  I think glslangValidator does allow it from version 1.50, so I probably just made an error.  Can you please use glslangValidator and also refer to the shading language versions to check what is the right thing to do here?

Reply:
You are right, glslangValidator now does allow this built-in function from version 1.50.